airbyte-integrations/connectors/source-google-ads/unit_tests/test_google_ads.py

```diff
@@ -75,10 +75,12 @@ def load_from_dict(config):
 
 
 SAMPLE_CONFIG = {
-    "developer_token": "developer_token",
-    "client_id": "client_id",
-    "client_secret": "client_secret",
-    "refresh_token": "refresh_token",
+    "credentials": {
+        "developer_token": "developer_token",
+        "client_id": "client_id",
+        "client_secret": "client_secret",
+        "refresh_token": "refresh_token",
+    },
     "customer_id": "customer_id",
```

Comment:
I guess start_date is required

Reply:
yes, but this test tests only connect for `GoogleAds` class,  and uses `SAMPLE_CONFIG` without `start_date` and `conversion_window_days`